### PR TITLE
Feature/soundsgoodtome

### PR DESCRIPTION
**Pull request by QuantumBits**
# Sound Features

* Press 'M' to play an A sound (440Hz)

### CODE REVIEW

File: src/common/input/input.ts [hunk 1]
```diff
@@ -49,6 +52,7 @@ export class Input {
   };
 
   private readonly onKeyEvent = (inputEvent: InputEvent) => {
+    logDebug(`KEY PRESSED : ${inputEvent.key}`);
```
Comment (alexkindel) [inline src/common/input/input.ts]: Remove log

Comment (QuantumBits) [reply src/common/input/input.ts]: but... but... fiiiiiine

File: src/common/audio/midi.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    this.an.connect(this.dn);
+    this.dn.connect(this.gn);
+
+    // this.cn.connect(this.gn);
+
+    this.ad = this.ac.destination;
+  } // tslint:enable:no-magic-numbers
+
+  public update(): void {
+    // TODO: Make music start/stop playing upon pressing "M"?
```
Comment (alexkindel) [inline src/common/audio/midi.ts]: Remove TODO if this feature is added

Comment (QuantumBits) [reply src/common/audio/midi.ts]: FIIIIINE

**Pull request merged**